actinia_stac_plugin/api/stac.py

```diff
@@ -65,7 +65,7 @@ def delete(self):
 
         return make_response(deleted_stac,200)
 
-class StacCollecions(ResourceBase):
+class StacCollections(ResourceBase):
```

Comment:
Do the and the following changes belong in this pull request?

Reply:
Not really, they are related to the commit right before, I would need to split the changes